src/infra/src/table/migration/m20250109_075148_alter_folders_id.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright 2024 OpenObserve Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Drops the folder's created_at column since Sea ORM maps this column's data
+//! type to different Rust types creating runtime errors when running on a
+//! non-PostgreSQL database.
+
+use config::ider;
+use sea_orm::{EntityTrait, PaginatorTrait, Set, TransactionTrait};
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        add_nullable_folder_uuid_column(manager).await?;
+        update_folder_uuid_column(manager).await?;
+        Ok(())
+    }
+
+    async fn down(&self, _manager: &SchemaManager) -> Result<(), DbErr> {
+        // Reversing this migration is not supported.
+        Ok(())
+    }
+}
+
+// Removes the old created_at column.
+async fn add_nullable_folder_uuid_column(manager: &SchemaManager<'_>) -> Result<(), DbErr> {
+    manager
+        .alter_table(
+            Table::alter()
+                .table(Folders::Table)
+                .add_column(
+                    ColumnDef::new(Folders::FolderUuid)
+                        .string_len(256)
```

Comment:
```suggestion
                    ColumnDef::new(Folders::FolderUuid)
                        .char_len(27)
```

These should always be 27-character fixed length strings so we can make the column type a bit more restrictive.

Reply:
Nit: I would recommend renaming this from `folder_uuid` to `folder_ksuid` just to make it more obvious that this should store a [ksuid](https://docs.rs/svix-ksuid/0.8.0/svix_ksuid/) rather than a [uuid](https://docs.rs/uuid/1.11.0/uuid/)